src/main/kotlin/org/jitsi/rtp/RtpHeaderExtensions.kt

```diff
@@ -40,8 +36,8 @@ class RtpHeaderExtensions {
         fun parse(buf: ByteBuffer): MutableMap<Int, RtpHeaderExtension> {
             val headerExtensionType = buf.getShort()
             val headerExtensionParser = when {
-                headerExtensionType.isOneByteHeaderType() -> ::RtpOneByteHeaderExtension
-                headerExtensionType.isTwoByteHeaderType() -> ::RtpTwoByteHeaderExtension
+                RtpOneByteHeaderExtension.headerExtensionTypeMatches(headerExtensionType) -> ::RtpOneByteHeaderExtension
```

Comment:
just wondering, why do you prefer this way over the extension method?

Reply:
I just wanted to keep the logic in the extension classes

Reply:
How about marking the extension functions as private?  That's probably what it should've been anyway (also, extension functions aren't applied globally: they have to be explicitly imported so we're not 'polluting' all `Short`s with those helper functions).

I think the static function is ok, it's just so much more verbose (and I do think the extension function reads more naturally)